### DISCUSSION TITLE
Has anyone tried SynoAI with Surveillance Station 9.0 beta?

### DISCUSSION THREAD

**Discussion by mkammes** [djdd87/SynoAI#114] [Q&A] [answered]
I'm eager to move off of SS 8.2 and wondering if any users have tested SynoAI with 9.0 beta. Disclaimer: I realize SynoAI is not officially supported with SS 9.0 beta.

Thanks!

**Comment by djdd87** [comment]
I haven't, no. And I'm unfortunately unlikely to get time too in the near future. It would be great if someone could test and report back.

**Comment by Neogx** [comment, marked as answer]
Im using it on 9.0 beta seems to be working okay
